appgate/attrs.py

```diff
@@ -228,6 +228,14 @@ def _attrload(l, value, type_):
         names = []
         defaults = {}
         types = {}
+
+        # Manage name mangling
+        namesmap: dict[str, str] = {}
+        for attribute in type_.__attrs_attrs__:
```

Comment:
can we avoid one for loop? Can you do the mangling inside the for after?

Reply:
The namesmap is created inside the same for loop. That's the data structure used to tell which names to rename, to the typedload function that does the renaming. 

Or what loop are you referring to?